RHIDP-11676: Orchestrator - Migrate doc for Log aggregation and Open Telemetry feature (#2027)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/extend_orchestrator-in-rhdh/assembly-identify-and-eliminate-workflow-bottlenecks-to-meet-slas.adoc

:_mod-docs-content-type: ASSEMBLY
ifdef::context[:parent-context: {context}]

[id="identify-and-eliminate-workflow-bottlenecks-to-meet-slas_{context}"]

> = Identify and eliminate workflow bottlenecks to meet SLAs

Reviewer: ```suggestion
= Optimize workflow performance by identifying and eliminating bottlenecks
```
(For a JTBD type of title. Just a suggestion)

:context: identify-and-eliminate-workflow-bottlenecks-to-meet-slas

[role="_abstract"]

> Visualize the complete execution path of workflows to pinpoint bottlenecks and meet SLA targets. Distributed tracing shows exactly how long each step takes and where time is spent across service boundaries.

Reviewer: ```suggestion
To maintain high performance in {product}, you must identify and resolve execution delays. Use distributed tracing to visualize the execution path and workflows and determine where time is spent across service boundaries.```

// Enable OpenTelemetry for SonataFlow workflows
include::../modules/extend_orchestrator-in-rhdh/proc-collect-traces-to-see-execution-timing-and-failures.adoc[leveloffset=+1]

> // Configuring telemetry exporters for Jaeger and Loki

Reviewer: ```suggestion
// Connecting traces to monitoring tools
```
